components/webac/src/main/java/org/trellisldp/webac/WebACService.java

```diff
@@ -73,6 +75,7 @@
  * @author acoburn
  */
 @Alternative
+@Priority(Interceptor.Priority.APPLICATION+10)
```

Comment:
Checkstyle issue: add whitespace around the `+` operator

Reply:
If there is any magic to 10 or not, it would be nice to have it broken out into a constant and given a comment to the correct effect.